src/js/index.js

```diff
@@ -0,0 +1,218 @@
+import CreatePage from './components/createHtml/createHtml';
+import * as board from './components/main/board';
+
+import '../css/main.scss';
+
+const createPage = new CreatePage();
+
+const LANG = { RU: 'ru', EN: 'en' };
+
+const state = {
+  lang: LANG.RU,
+  capsLock: false,
+};
+
+const KEYBOARD = {
+  tab: '  ',
+  space: ' ',
+  enter: '\n',
+};
+
+window.addEventListener('beforeunload', () => {
+  localStorage.setItem('language', state.lang);
+});
+
+window.addEventListener('DOMContentLoaded', () => {
+  const language = localStorage.getItem('language');
+  if (language !== null) {
+    state.lang = language;
+  }
+});
+
+window.addEventListener('DOMContentLoaded', () => {
+  createPage.create();
+  board.createKeyBoard(state.lang);
+
+  const keyBoard = document.querySelector('.keyboard-list');
+
+  document.addEventListener('keydown', (event) => {
+    const switcherCtrlAlt = event.ctrlKey && event.altKey;
+    const switcherShiftAlt = event.shiftKey && event.altKey;
+    if (switcherCtrlAlt || switcherShiftAlt) {
+      state.lang = state.lang === LANG.RU ? LANG.EN : LANG.RU;
+      keyBoard.innerHTML = '';
+      keyBoard.innerHTML = board.generateButtons(state.lang);
+    }
+  });
+
+  const textarea = document.querySelector('.textarea');
+  const capsLock = document.querySelector('.capslock');
+
+  const capsLockHandler = () => {
+    const buttons = document.querySelectorAll('.keyboard__button');
+    if (!state.capsLock) {
```

Comment:
There's no need in if else statement here, you can write it in a single code that will execute for all buttons and perform different actions depending on caps lock state.

Reply:
Created new capsLock function